esp/bindings/http/platform/httptransport.cpp

```diff
@@ -2541,3 +2634,182 @@ void CHttpResponse::handleExceptions(IXslProcessor *xslp, IMultiException *me, c
     if (handleExceptions(xslp, me, serv, meth, errorXslt) && logHandleExceptions)
         PROGLOG("Exception(s) handled");
 }
+
+inline bool zlibTypeFromHeader(const char* acceptEncodingHeader, int& compressType)
```

Comment:
Don't know how often it's used, but there is the possiblity of weights after ;
Accept-Encoding: br;q=1.0, gzip;q=0.8, *;q=0.1
May at least want to strip them off before comparisons.